src/main/scala/com/wavesplatform/state2/diffs/ReleaseSlotsTransactionDiff.scala

```diff
@@ -38,7 +38,9 @@ object ReleaseSlotsTransactionDiff {
     else if (hasEnoughMiner && isValidAddress && isValidSlotID) {
       Right(Diff(height = height, tx = tx,
         portfolios = Map(sender.toAddress -> Portfolio(-tx.fee, LeaseInfo.empty, Map.empty)),
-        slotids = Map(tx.slotId -> emptyAddress),slotNum = -1, chargedFee = tx.fee))
+        slotids = Map(tx.slotId -> emptyAddress),
+        addToSlot = Map(sender.toAddress.address -> -1),
```

Comment:
[IMO] -1 here may cause some confusion. What about using Option[Int], and None for here.

Reply:
since, slotids is a `Map[String, Int]` and `address->None` is `Map[String, Any]`
when using `++`, it may cause some issue

```scala
scala> val x = Map(1 -> 10)
x: scala.collection.immutable.Map[Int,Int] = Map(1 -> 10)

scala> val y: Map[Int, Int] = x ++ Map(1 -> None)
<console>:12: error: type mismatch;
 found   : scala.collection.immutable.Map[Int,Any]
 required: Map[Int,Int]
       val y: Map[Int, Int] = x ++ Map(1 -> None)
                                ^
```

Reply:
I mean using ```Map[String, Option[Int]]``` instead of ```Map[String, Int]```

Reply:
i see, but it will need `Option()` in all related maps, ^^.  e.g.
```scala
scala> val x = Map(1->Option(10))
x: scala.collection.immutable.Map[Int,Option[Int]] = Map(1 -> Some(10))

scala> val y: Map[Int, Option[Int]] = x ++ Map(1->None)
y: Map[Int,Option[Int]] = Map(1 -> None)
```

Reply:
yes, it may cause some complexity. 
```Map(sender.toAddress.address -> tx.slotId)``` -> ```Map(sender.toAddress.address -> Some(tx.slotId))```
and ```Map.get``` will return a ```Option[Option[Int]]```